packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -150,15 +157,41 @@ const overviewContent = (
     <Grid item md={6}>
       <EntityAboutCard variant="gridItem" />
     </Grid>
-    <Grid item md={6}>
-      <LambdaFunctionCard />
-    </Grid>
-    <Grid item md={6}>
-      <S3BucketCard />
-    </Grid>
-    <Grid item md={6}>
-      <IAMUserCard />
-    </Grid>
+    <EntitySwitch>
```

Comment:
So basically this all adds frontend cards to the entity page, if the right annotations are present on the entity.